packit_service/worker/handler.py

```diff
@@ -168,10 +168,11 @@ def _clean_workplace(self):
             logger.info("volume is not empty")
             logger.debug("content: %s" % [g.name for g in dir_items])
         for item in dir_items:
-            if item.is_dir():
-                shutil.rmtree(item)
-            else:
+            # symlink pointing to a dir is also a dir and a symlink; please stop changing this code
```

Comment:
also [PullRequestCommentHandler._clean_workplace()](https://github.com/packit-service/packit-service/blob/master/packit_service/worker/pr_comment_handler.py#L100)

Reply:
Actually [unlink() doc](https://docs.python.org/3/library/pathlib.html#pathlib.Path.unlink) says that "If the path points to a directory, use Path.rmdir() instead.".

So my question is what we actually want to do in case of symlink to dir?
a) `unlink()` -> `rmdir()` -> fail if dir not empty
b) `rmtree()`